streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -78,6 +78,7 @@ public void before() {
         props.setProperty(StreamsConfig.METRIC_REPORTER_CLASSES_CONFIG, MockMetricsReporter.class.getName());
         props.setProperty(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getPath());
         props.put(StreamsConfig.NUM_STREAM_THREADS_CONFIG, NUM_THREADS);
+        props.put(StreamsConfig.RETRIES_CONFIG, Integer.MAX_VALUE);
```

Comment:
Why increase the retries here on in the `EosIntegrationTest`?

Reply:
Can't remember :) Maybe a merge error on rebase or I tried something out. Reverting this.